contrib/libvirt/files/virtnwfilterd

```diff
@@ -0,0 +1,9 @@
+# virtnwfilterd daemon service
+# clonficts with libvirtd daemon
+
+type          = process
+command       = /usr/bin/virtnwfilterd
+depends-on    = network.target
+after         = dbus
```

Comment:
idk if these request a name on the bus, but if they do, they should be `depends-on = dbus`

Reply:
i do not know how to figure that out, I'll leave it as is for now